[2025-04-20T17:34:15.304+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-04-20T17:34:15.370+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: supply_chain_dag.collect_data manual__2025-04-20T17:34:12.387448+00:00 [queued]>
[2025-04-20T17:34:15.397+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: supply_chain_dag.collect_data manual__2025-04-20T17:34:12.387448+00:00 [queued]>
[2025-04-20T17:34:15.397+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-04-20T17:34:15.433+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): collect_data> on 2025-04-20 17:34:12.387448+00:00
[2025-04-20T17:34:15.442+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=974) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-04-20T17:34:15.446+0000] {standard_task_runner.py:72} INFO - Started process 975 to run task
[2025-04-20T17:34:15.446+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'supply_chain_dag', 'collect_data', 'manual__2025-04-20T17:34:12.387448+00:00', '--job-id', '4', '--raw', '--subdir', 'DAGS_FOLDER/supply_chain_dag.py', '--cfg-path', '/tmp/tmp03pmv2bn']
[2025-04-20T17:34:15.448+0000] {standard_task_runner.py:105} INFO - Job 4: Subtask collect_data
[2025-04-20T17:34:15.526+0000] {task_command.py:467} INFO - Running <TaskInstance: supply_chain_dag.collect_data manual__2025-04-20T17:34:12.387448+00:00 [running]> on host 6d6cd65cf27d
[2025-04-20T17:34:15.694+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='supply_chain_dag' AIRFLOW_CTX_TASK_ID='collect_data' AIRFLOW_CTX_EXECUTION_DATE='2025-04-20T17:34:12.387448+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-20T17:34:12.387448+00:00'
[2025-04-20T17:34:15.696+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-04-20T17:34:15.810+0000] {logging_mixin.py:190} INFO - {'status': 'Data sent to Kafka', 'data': {'order_id': 'ORD-5AE074A8', 'departure_date': '2025-04-17', 'arrival_date': '2025-04-25', 'quantity': 331, 'warehouse': 'WH3', 'destination': 'DistributionCenter', 'shipping_mode': 'Standard Class', 'order_amount': 2458.33, 'customer_id': 'CUST-8598FF33', 'late_delivery_risk': 0}}
[2025-04-20T17:34:15.812+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-04-20T17:34:15.829+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-04-20T17:34:15.831+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=supply_chain_dag, task_id=collect_data, run_id=manual__2025-04-20T17:34:12.387448+00:00, execution_date=20250420T173412, start_date=20250420T173415, end_date=20250420T173415
[2025-04-20T17:34:15.950+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-04-20T17:34:15.986+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-04-20T17:34:15.988+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
